src/resources/SecurityCache/SecurityCacheInterfaces.ts

```diff
@@ -0,0 +1,100 @@
+export interface SecurityCacheScheduleModel {
+    enabled?: boolean;
+    frequency?: {
+        hourMinute?: {
+            hour?: number;
+            minute?: number;
+        };
+        monthDay?: number;
+        seed?: number;
+        type?: string;
+        value?: number;
+        weekDays?: [string];
+    };
+    id?: string;
+    organizationId?: string;
+    refreshType?: string;
+    resourceId?: string;
+    scheduleType?: string;
+}
+
+interface Status {
+    numberOfEntitiesInError?: number;
+    numberOfEntitiesProcessed?: number;
+    refreshType?: string;
+    totalNumberOfEntities?: number;
+    type?: string;
+}
+
+interface Statistics {
+    numberOfEntitiesByState?: {};
+    numberOfEntitiesInError?: number;
+    numberOfPermissionModels?: number;
+    numberOfProviders?: number;
+    totalNumberOfEntities?: number;
+}
+
+export interface SecurityCacheStatus {
+    currentStatus?: Status;
+    statistics?: Statistics;
+}
+
+export interface SecurityCacheIdentityModel {
+    data: string;
+    provider: string;
+}
+
+export interface DetailedSecurityCacheMemberModel {
+    infos?: [{key?: string; value?: string}];
+    lastUpdateDate?: number;
+    lastUpdateErrorDetail?: string;
+    lastUpdateResult?: string;
+    name?: string;
+    provider?: string;
+    state?: string;
+    type?: string;
+}
+
+export interface SecurityProviderModel {
+    cascadingSecurityProviders: {};
+    caseSensitive: boolean;
+    currentStatus: Status;
+    displayName: string;
+    id: string;
+    instanceId: string;
+    lastRefreshOperation: {
+        errorCode: string;
+        id: string;
+        numberOfEntitiesProcessed: number;
+        operationType: string;
+        result: string;
+        timestamp: number;
+    };
+    name: string;
+    nodeRequired: boolean;
+    nodeTypeName: string;
+    organizationId: string;
+    parameters: {};
+    referencedBy: [
+        {
+            attributes: {};
+            id: string;
+            type: string;
+        }
+    ];
+    sourceTypeName: string;
+    statistics: Statistics;
+    type: string;
+    userIdentities: {};
+}
+
+export interface SecurityCacheModelOptions {
+    from?: number;
+    organisationId: string;
+    page?: number;
+    perPage?: number;
+    securityProviderId: string;
+    states: string[];
```

Comment:
is there an enum defined for this?

Reply:
:see_no_evil: 
Yeah and I'm using it. I forgot to remove this one !